handbook/engineering/distribution/goals.md

```diff
@@ -8,7 +8,7 @@ Goals are continuously updated and reviewed. If you find these goals do not refl
 
 Creating a new release for our deployments is currently a semi-automated process, which requires several manual steps and synchronizing our versioned artifacts (Sourcegraph, Kubernetes manifests, docker-compose manifests, etc). We want to enable any engineer to perform a release as often as needed, to enable this we want to make releasing Sourcegraph a simple, automated process.
 
-- **Owner**: Stephen, Dave, Robert
+- **Owner**: Dave, Robert
 - **Status**: In progress. Estimated completion by end of 2020.
```

Comment:
@pecigonzalo I think this may change the status of this goal, I suspect we will get pretty close by EOY but not exactly 100% of the way there. Maybe like 70-80% of the way there. Thoughts on how to update this to reflect that? And do you agree with that assessment?

Reply:
We can swap that to FY21-Q1, I think that should give us enough buffer for it. @davejrt @bobheadxi do you think that is accurate?